accessible/workspaceview.component.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Blockly Demos: AccessibleBlockly
```

Comment:
This is not a demo, I'd replace this line with:
 * Accessible Blockly
And all similar files.